recipes/ratarmountcore/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "ratarmountcore" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ratarmountcore-{{ version }}.tar.gz
+  sha256: 320c87813aae4137881ab26641efb65d6b618ced002e2550787c1c2c2f851a53
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - dataclasses  # [py<37]
+    - importlib-metadata  # [py<38]
```

For noarch (we can just unconditionally depend on the other packages):
```suggestion
    - python >=3.6
    - pip
  run:
    - python >=3.6
    - dataclasses
    - importlib-metadata
```